src/components/Display.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export, react/prop-types */
```

Comment:
Please don't disable these checks, as they are easily fixable. Simply export default this function, since it's the only one in the file, and also please validate the props you are passing with `prop-types`